packages/rancher-gatekeeper/rancher-gatekeeper.patch

```diff
@@ -259,34 +1148,33 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper/charts-original/tem
 +          def_msg := sprintf("Label <%v: %v> does not satisfy allowed regex: %v", [key, value, expected.allowedRegex])
 +          msg := get_message(input.parameters, def_msg)
 +        }
-\ No newline at end of file
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper/charts-original/values.yaml packages/rancher-gatekeeper/charts/values.yaml
 --- packages/rancher-gatekeeper/charts-original/values.yaml
 +++ packages/rancher-gatekeeper/charts/values.yaml
-@@ -1,12 +1,12 @@
- replicas: 1
+@@ -1,5 +1,5 @@
+ replicas: 3
 -auditInterval: 60
 +auditInterval: 300
  constraintViolationsLimit: 20
  auditFromCache: false
+ disableValidatingWebhook: false
```

Comment:
Is this what got disabled ? Here it looks like you are re-enabling it.

Reply:
That was added as part of the patch and I missed it. However, thanks for pointing it out because I could just disable the webhook validation from here and keep the yaml which will clean up the patch a bit by not showing the deletion.

Reply:
That sounds cleaner.